internal/i18n/locales/id/default.po

```diff
@@ -0,0 +1,228 @@
+msgid ""
+msgstr ""
+"Language: id\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+
+#
+# navigation menus
+# ----------------
+
+msgid "nav.issue-code"
+msgstr "Kode masalah"
```

Comment:
Hey @dwisiswant0 thank you for the PR. I think this translation might be incorrect. "Issue" is a verb in this case (synonyms: create, generate, make). I think this translation to "trouble".

Reply:
(For all instances of "issue"). I'm not a native speaker though, so please correct me if I'm wrong

Reply:
Thanks for review this, looks like your context is correct. Should be resolved as per 63b4fee.